hack/testing-olm-upgrade/test-020-olm-upgrade-4.4-4.5.sh

```diff
@@ -66,6 +66,13 @@ get_es_indices() {
     | xargs -I '{}' oc -n openshift-operators-redhat exec '{}' -c elasticsearch -- es_util --query=_cat/indices
 }
 
+get_es_indices_names() {
+  oc -n openshift-operators-redhat get pods -l component=elasticsearch --no-headers=true --ignore-not-found \
```

Comment:
we don't have ES pods deployed in this namespace at all. the operator is in `openshift-operators-redhat`

Reply:
In general, we don't, but this test deploys EO into that namespace for simplicity.